src/cloud-api-adaptor/install/overlays/aws/cri_runtime_endpoint.yaml

```diff
@@ -1,15 +1,15 @@
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: cloud-api-adaptor-daemonset
```

Comment:
You are deleting some files with 8338cb70f29058b32871ff6547699a5a61102142, so I guess part of this commit could be dropped.

Reply:
Yeah - I was waiting for confirmation that we are fine to delete the cri_runtime_endpoint and then I'd re-order, so that that change happens first.

Reply:
The cri_runtime_endpoint kustomize delete has been moved above this commit in the stack, so this code is gone now.